frontend/packages/kubevirt-plugin/src/selectors/selectors.ts

```diff
@@ -12,6 +13,8 @@ export const getAnnotations = (vm: VMLikeEntityKind, defaultValue?: any) =>
 export const getDescription = (vm: VMLikeEntityKind) =>
   _.get(vm, 'metadata.annotations.description');
 
+export const getCdRom = (vm: VMLikeEntityKind) => getDisks(vm).filter((device) => !!device.cdrom);
```

Comment:
ok, but still, we should call this function `getCDRoms` (plural and `CD` capital letters).

Reply:
fixed